_includes/footer.html

```diff
@@ -0,0 +1,49 @@
+<footer class="site-footer">
```

Comment:
Is addition of this new file part of "Add link to github repo of the website"? If not, it should go in a separate patch.